frontend/package.json

```diff
@@ -67,6 +67,7 @@
     "prebuild": "yarn compile-scss",
     "build": "INLINE_RUNTIME_CHUNK=false REACT_APP_CURRENT_COMMIT=$(git rev-parse HEAD) craco build",
     "postbuild": "echo 'pushstate: enabled' > build/Staticfile && echo $(git rev-parse HEAD) > build/static/commit.txt",
+    "codegen": "graphql-codegen --config codegen.yml",
```

Comment:
is this something we have to run manually everytime we change the graphql schema?

Reply:
This is a really good flag. Thank you! It was going to be manual. Added a commit after seeing this to watch for changes and automatically update the codegen https://github.com/CDCgov/prime-simplereport/pull/2556/commits/bbbc4c366cca311b77490944055c8b66e50557c9